Migration to 0.8.0

jsperezg opened the pull request:
Added short migration guide for Decidim 0.8.0.

#### :tophat: What? Why?
It adds a small migration guide with the issues that I've found while migrating an engine to this new version. I'm pretty sure that it's not finished yet but it will help other users to migrate for sure.

xabier commented:
nice one @jsperezg ! thanks!

josepjaume commented on docs/migrate_to_0.8.0.md:
```diff
@@ -0,0 +1,111 @@
+
+## Note about this guide.
+
+This is a work-in-progress guide for all those people that needs to adapt his existing source code to Decidim 0.8.0. If you find a mistake or missing parts in this  document do not hesitate to make a pull request and add your discoveries.
```
I would specify: "From 0.7.0" to "0.8.0"

josepjaume commented on docs/migrate_to_0.8.0.md:
````diff
@@ -0,0 +1,111 @@
+That was caused by a confirmation dialog. In order to get rid of these issue I had to add the following line in the point where the dialog was supposed to be accepted:
+
+```ruby
+page.driver.browser.switch_to.alert.accept
````
You should use `accept_confirm` as in https://github.com/decidim/decidim/blob/9d136b8d66d0bd4e3e8449002c1dd3a1a45c0a53/decidim-surveys/spec/features/survey_spec.rb#L84 instead.

josepjaume commented on docs/migrate_to_0.8.0.md:
````diff
@@ -0,0 +1,111 @@
+
+```bash
+$ cd spec/decidim_dummy_app
+$ bundle exec rails db:drop
````
You should probably copy the new migrations as well: `rake decidim:upgrade`.

josepjaume commented on docs/migrate_to_0.8.0.md:
````diff
@@ -0,0 +1,111 @@
+$ bundle exec rails db:seed
+```
+
+Finally, take a cold beer and enjoy democracy.
````
🍻 😄

josepjaume commented:
@jsperezg could you change this so I can 🚢  it?

jsperezg commented:
@josepjaume Done.

I've fixed all your comments. Additionally I've added some information regarding Uglifier and ES6 support in production environments.

mrcasals commented:
Merging before running the tests, as we're only modifying Markdown files.

mrcasals commented:
Yay @jsperezg, thank you very much! ❤️
